CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Draft
 - Use appropriately-sized (50x50) images for product thumbnails on product details page [#1097](https://github.com/bigcommerce/cornerstone/pull/1097)
+- Load visible images before images below the fold [#1104](https://github.com/bigcommerce/cornerstone/pull/1104)
+- Save space for lazy loading images while they are loading [#1104](https://github.com/bigcommerce/cornerstone/pull/1104)
```

Comment:
I am not sure we should be adding multiple lines pointing to the same PR. Can we update the desc in the first one to include the second one as well.